.github/workflows/publish-dev.yml

```diff
@@ -51,13 +51,22 @@ jobs:
           node-version: 18
           registry-url: https://registry.npmjs.org/
 
+      - name: Check previous released version
+        id: pre-release
+        run: |
+          if [[ $(npm view ${{ matrix.package }}@dev version | grep -e "$(jq --raw-output '.version' packages/${{ matrix.folder }}/package.json)\..*-$(git rev-parse --short HEAD)") ]]; \
```

Comment:
If I follow the steps for discord.js right now:

1. `npm view discord.js@dev version | grep -e "$(jq --raw-output '.version' packages/discord.js/package.json)\..*-$(git rev-parse --short HEAD)")`
2. `14.14.0-dev.1695168548-1e5c14b74.0 | grep -e "14.13.0\..*-1e5c14b74"`

This seems like it would fail... but it should have passed. Perhaps looking for `-$(git rev-parse --short HEAD)` would be enough? Or, am I missing something?